toolz/itertoolz.py

```diff
@@ -15,6 +14,9 @@
            'join', 'tail', 'diff', 'topk', 'peek')
 
 
+_no_default = object()  # sentinel
```

Comment:
I have at times needed to explicitly state `__no_default__` as an argument to functions.  In these cases it is nice when `toolz` and `cytoolz` share a common sentinel value.